stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -0,0 +1,233 @@
+import collections.abc
+from _typeshed import Incomplete
+from collections.abc import Callable, Generator
+from datetime import datetime
+from typing import NamedTuple, TypeVar, overload
+from typing_extensions import ParamSpec
+
+# from pyscreeze import Box
+
+class PyAutoGUIException(Exception): ...
+class FailSafeException(PyAutoGUIException): ...
+class ImageNotFoundException(PyAutoGUIException): ...
+
+collectionsSequence = collections.abc.Sequence
+
+_P = ParamSpec("_P")
+_R = TypeVar("_R")
+
+# TODO: Complete types with pyscreeze once we can import non-types dependencies
+# See: https://github.com/python/typeshed/issues/5768
+
+class _Box(NamedTuple):
```

Comment:
This looks correct, but it would be good to add a comment here explaining where `_Box` is actually defined. I had to do some hunting to find the definition!

```suggestion
# Actually `pyscreeze.Box`, but typeshed doesn't currently have stubs for pyscreeze
# (and the library doesn't have type annotations either)
class _Box(NamedTuple):
```